### ISSUE TITLE
EPIC: Data Site Homepage

### ISSUE THREAD

**Issue by jjoteal** [RTICWDT/college-scorecard#266]
Desktop reference comp:
![image](https://cloud.githubusercontent.com/assets/12192706/8983994/8e4fc4f8-369c-11e5-8557-d7481468ca85.png)

Mobile reference comp:
![image](https://cloud.githubusercontent.com/assets/12192706/8983998/98b60a10-369c-11e5-8469-2c3750ea3b05.png)

Issues for homepage sections:
1. HP Top Text Intro Blurb - #143
2. Top Nav – #263
3. API Documentation block - #161
4. Download the Data section with just the Download All Data button. (Create custom data set button should be a separate story as I don’t know if that will make it into 1.0. Featured Downloads should also be a separate story) - #160. New issue for the “Create Custom Data Set” button - #262. New issue for "Featured Downloads - #268.
5. Data Insights - #157
6. Data At A Glance – Jess – I’d love to review functionality we anticipate here- #159. These would just be expand collapse sections. The title of the unit would be the Data Category Name (e.g. Earnings) and the information contained in the expanded container would be a list of  the individual metrics (i.e. average salary 10 years after leaving school) in plain English.
7. See the Data in Action – #265. Please note that I need to create a mobile comp for this. This feature will likely NOT  be included in the first release.
8. Join the Conversation – #264. Please note that I need to create a mobile comp for this. Not sure if this feature will be included in the first release?
9. Featured Downloads – #268.
10. Create Custom Data Set – Per above new issues #262
11. Layman’s Video (deferred from 1.0) - #158

**Comment by meiqimichelle** [comment]
Thanks for this @jjoteal -- super, super helpful!

**Comment by LisaGee** [comment]
@barkimedes We should look at each of the substories and see what still needs to be done and then perhaps create a new story and retire this one.

**Comment by brownpl** [comment]
Retiring this story. Will create new stories for updates to the data homepage.
